src/api/app/jobs/staging_project_copy_job.rb

```diff
@@ -0,0 +1,7 @@
+class StagingProjectCopyJob < ApplicationJob
+  def perform(staging_workflow_project_name, original_staging_project_name, staging_project_copy_name)
```

Comment:
Nitpick: I would think about shortening the variable names.

Reply:
I agree that they are quite long, but I didn't find something shorter without losing clarity.

Reply:
How about this?:

staging_workflow_project_name -> workflow_name
original_staging_project_name -> original_name
staging_project_copy_name -> copy_name

Reply:
Hum... I prefer longer names, they are clearer. If someone else thinks we should go for the names you suggest, I'm fine with changing them.